voting-hir/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use jormungandr_lib::{crypto::account::Identifier, interfaces::Value};
+
+pub type VotingGroup = String;
+
+/// Define High Level Intermediate Representation (HIR) for voting
+/// entities in the Catalyst ecosystem.
+///
+/// This is intended as a high level description of the setup, which is not
+/// enough on its own to spin a blockchain, but it's slimmer, easier to understand
+/// and free from implementation constraints.
+///
+/// You can roughly read this as
+/// "voting_key will participate in this voting round with role voting_group and will have voting_power influence"
+pub struct VoterHIR {
```

Comment:
are we going to implement serde traits for this?

Reply:
Yes, even though we may want to serialize the voting key differently from the default bech32 encoding (e.g. hex). Maybe I should add some helper methods for that here